src/yb/yql/pggate/pg_client.cc

```diff
@@ -440,7 +440,8 @@ class PgClient::Impl {
     *req.mutable_options() = std::move(*options);
     PrepareOperations(&req, operations);
     // Amit: This is probably unnecessary. header_ already contains a call-id.
-    req.mutable_options()->mutable_auh_metadata()->set_current_request_id(reinterpret_cast<uint64_t>(&req));
+    // Abhinab: Using the header call id as the request id.
+    // req.mutable_options()->mutable_auh_metadata()->set_current_request_id(reinterpret_cast<uint64_t>(&req));
```

Comment:
Why is this commented?

Reply:
There seems to be only 2 addresses which get reused everytime, so that means the request id also stays the same even with different requests. Also, when the call reaches tserver, we are using header.call_id anyway.

Reply:
Probably remove lines 442-444?